TAC/Meetings/2024/CCC-TAC-Minutes-2024-01-11.md

```diff
@@ -0,0 +1,126 @@
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+## Agenda and Minutes
+
+### Welcome
+
+* Dan Middleton (DM) opened the call at 7:05am PT
+* DM welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+* Ben Sternthal (BS) and Riaan Kleinhans (RK) assisted with the meeting minutes
+* DM reviewed the agenda
+
+### Roll Call / Attendance
+
+Quorum was not achieved.
+
+#### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* Arm - Nathaniel McCallum  / Michael Lu
+* Google - Cfir Cohen  / Catalin Sandu +
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair) + / Simon Johnson
+* Meta - Eric Northup / Shankaran Gnanashanmugam
+* Microsoft - Alec Fernandez 
+* Red Hat - Lily Sturmann  / Yash Mankad +
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Chris Ramming (Broadcom)
+* Dan Willians (Intel)
+* Drasko Draskovic (Ultraviolet)
+* Haitao Huang
+* Hesham ElBakoury (Nortel)
+* Ijlal Loutfi (Canonical)
+* Jorg Rodel (SUSE)
+* Manu Fontaine (Hushmesh)
+* Mark Novak (JP Morgan Chase)
+* Ofir Azoulay-Rozanes (Anjuna)
+* Samuel Ortiz (Rivos)
+* Thomas Fossati (Linaro)
+
+* Ben Sternthal (LF)
+* Mike Bursell (CCC)
+* Riaan Kleinhans (LF)
+* Sal Kimmich (LF)
+
+
+
+## Welcome New Community Members
+
+* Sal Kimmich introduced themselves as the new Technical Community Architect
+* Ljal Loutfi from Canonical introduced themselves
+
+## Meeting Minute Approval & Old Business
+
+* Meeting minutes will be approved via Github
+* Dan Middleton (DM) provided an update on Coconut-svsm and the Kernel SIG
+* DM discussed project mentors and options for scaling the TAC
+
+## Tech Talk: Linux Kernel Upstream and our new SIG
+
+Dan Williams from Intel presented. General discussion ensued.
+Action item: Riaan - set up the SIG on CCC website with links
+
+## New Business
+
+* Mike Bursell (MB) proposed adjusting meeting times to accommodate timezones in APAC and that we could have a meeting time that alternates between timezones.
+* Sal Kimmich pointed out that voting could be a challenge. They proposed 3PM GMT or 9AM GMT
+* Offir Azoulay-Rozanes (OAR) requested that a voting poll should be sent out
+* Dan Middleton (DM) suggested that we start with a 1 to 4 ratio for the APAC meeting
+
+## 2024 TAC Objectives
+
+Dan Middleton (DM)  introduced an OKR document and encouraged TAC Voting members to participate. (DM) will reach out to the voting members regarding the document.
+
+### RFI’s
+
```

Comment:
```suggestion

Discuss NIST documents per mail list thread https://lists.confidentialcomputing.io/g/tac/message/1254
Due January 15: NIST SP 1800-28 (Data Confidentiality: Identifying and Protecting Assets Against Data Breaches)
Due January 25: NIST SP 800-226 (Guidelines for Evaluating Differential Privacy Guarantees)
Due February 2: Safe, Secure, and Trustworthy Development and Use of AI
```
Added detail to understand which docs we were covering

Reply:
Updated. Thanks you @dcmiddle